internal/ethapi/api.go

```diff
@@ -1583,6 +1584,10 @@ func newRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNumber
 		result.PaymasterGas = (hexutil.Uint64)(tx.PaymasterGas())
 		result.PostOpGas = (hexutil.Uint64)(tx.PostOpGas())
 		result.BigNonce = (*hexutil.Big)(tx.BigNonce())
+	case types.Rip7560BundleHeaderType:
+		log.Info("rip7560(bundleHeader) new RpcTransaction")
+		result.ChainID = (*hexutil.Big)(tx.ChainId())
+		result.TransactionCount = (hexutil.Uint64)(tx.TransactionCount())
```

Comment:
Do we need changes here? This is not a tx crafted or received by RPC, isn't it?

Reply:
Required to configure responses to RPC requests. 
For example, this part is needed when serializing to an RPC representation format for the response of a request like getBlockByHash.